SingularityUI/app/templates/vex/requestRun.hbs

```diff
@@ -1,3 +1,14 @@
 <p>Are you sure you want to run a task for this request immediately?</p>
 <pre>{{ id }}</pre>
-<p>Additional command line input (optional):</p>
+
+<p>Additional command line input <small>(optional)</small>:</p>
+<div class="vex-dialog-input">
+	<input id='commandLineInput' name="commandLineInput" type="text" class="vex-dialog-prompt-input" placeholder="" value="">
+</div>
+<hr style='border-color: #dadada'>
+
+<p>Wait for task to start, then bring me directly to a file <small>(optional)</small>:</p>
```

Comment:
lets make this a little more understandable for someone who's never used this before (i.e. leave blank for no action)